README.md

```diff
@@ -64,6 +64,20 @@ FactoryHelper::String.random([0, 6]) #=> "I轤𣴒P溟L"
 ```
 
 
+###FactoryHelper::MySQL
+-----------------
+Generates values for a factory per the MySQL datatypes.  Accepts options unsigned, min, max.
+
+```ruby
+FactoryHelper::MySQL.integer #=> 1729
```

Comment:
This is part of the MySQL specs but we can add it for clarity.  I was deciding whether to include the note or not.